Makefile

```
@@ -135,6 +135,13 @@ device-trees: $(DESTDIR)/boot-assets
 	cp -a $$(find $(STAGEDIR)/lib/firmware/*/device-tree -name "*.dtbo") \
 		$(DESTDIR)/boot-assets/overlays/
 
+# ubuntu-image on classic does not understand the "$kernel:ref" syntax
+# yet and will most likely only do so once it is ported to golang. Use
+# the old syntax for now
+no-kernel-refs-gadget: gadget.yaml
+	mkdir -p $(DESTDIR)/meta
+	sed -e '/source: $kernel:/,+1d' gadget.yaml > $(DESTDIR)/meta/gadget.yaml
```

```suggestion
	sed -e '/source: $$kernel:/,+1d' gadget.yaml > $(DESTDIR)/meta/gadget.yaml
```

I think this won't work otherwise because it's part of a Makefile.